src/libs/actions/PersonalDetails.js

```diff
@@ -42,8 +42,8 @@ function getDefaultAvatar(login = '') {
  * @returns {String}
  */
 function getAvatar(personalDetail, login) {
-    if (personalDetail && personalDetail.avatar) {
-        return personalDetail.avatar.replace(/&d=404$/, '');
+    if (personalDetail.avatarThumbnail) {
```

Comment:
What do you think about keeping the null / undefined `personalDetail` check here just to be safe?

Reply:
I can add it back but ideally it should always exist, and there is likely another issue if it doesn't